packages/experiments/src/components/Chiclet/examples/Chiclet.Preview.Example.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { Chiclet, ChicletSize } from '@uifabric/experiments/lib/Chiclet';
+
+const SAMPLE_URL = 'https://contoso.sharepoint.com';
+
+export const ChicletPreviewExample: React.FunctionComponent<{}> = () => {
+  const Prev: React.FunctionComponent<{ className: string }> = props => {
+    const { className } = props;
+
+    return <img src="https://kbdevstorage1.blob.core.windows.net/asset-blobs/19666_en_1" className={className} />;
```

Comment:
I wonder if maybe we want to use something here that we know won't go away. We could use `http://placehold.it/100x100` or something similar—it looks like we use those in our [Image](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Image/examples/Image.Default.Example.tsx#L15) examples. This could potentially be fine though! What do you think @KevinTCoughlin?